packages/react-native-tracer-provider/android/gradle.properties

```diff
@@ -0,0 +1,5 @@
+TracerProvider_kotlinVersion=1.7.0
```

Comment:
is there a reason why we need a prefix here? `TracerProvider_`

Reply:
I had copied and pasted this from another example, I believe the reasoning is to distinguish it from another "kotlinVersion" property that may have been defined by the app that uses this package, e.g. in `packages/react-native-tracer-provider/android/build.gradle` we have:

```
def kotlin_version = rootProject.ext.has("kotlinVersion") ? rootProject.ext.get("kotlinVersion") : project.properties["TracerProvider_kotlinVersion"]
```

So we use a bare "kotlinVersion" property if it's available and then only fallback to this "TracerProvider_kotlinVersion" if it's not